### PR TITLE
Add a small script to change version

### PR DESCRIPTION
**Pull request by bhack**
In the meantime that we centralize/refactor the versioning this script could be helpful.

`./update_release_version.sh 2.2.0rc5`

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
I'm not sure I understand, what's the benefit of running 
```
./update_release_version.sh 2.2.0rc5
```
compared to opening Pycharm, VSCode, vim... and do 
```
Ctrl+Shift+R (or equivalent) 2.2.0rc4 -> 2.2.0rc5
```

**Comment by bhack** [comment]
Until we need to track multiple files to pilot the version I think it is easier to have single script in the root without opening an ide, find and replace, saving all the file in the editor, etc.

**Comment by gabrieldemarmiesse** [comment]
I'm not sure the complexity of this script and regex is worth the saving of the few seconds needed to open and IDE/text editor. I'm not a fan of maintaining a functionality present already in other tools. But maybe it's just me. If another maintainer is interested, we can keep this script.

**Comment by seanpmorgan** [comment]
> I'm not sure I understand, what's the benefit of running
> 
> ```
> ./update_release_version.sh 2.2.0rc5
> ```
> 
> compared to opening Pycharm, VSCode, vim... and do
> 
> ```
> Ctrl+Shift+R (or equivalent) 2.2.0rc4 -> 2.2.0rc5
> ```

So I think the replace strategy is safe to do for specific versions like `2.2.0rc4` becoming `2.2.0rc5`. However, for stable release updates, running CTRL+SHIFT+R is susceptible to make replacements in docs strings and other places where versioning is being discussed but shouldn't be updated.

File: update_release_version.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sed -ri "s/(TF_VERSION=|tensorflow(-cpu)*(~|=)=|tf-version: \[')[0-9]+[a-zA-Z0-9_.-]+/\1$1/g" \
```
Comment (seanpmorgan) [inline update_release_version.sh:1]: Please add copyright info at the top, and relocate to the `tools/` directory to keep project root clean.

**Comment by bhack** [comment]
@seanpmorgan As you can see the list of files is constrained but please double check the safety of the regular expression.

File: update_release_version.sh (hunk 1 above)
Comment (bhack) [reply update_release_version.sh]: I moved it to `tools/` but I don't see copyright info in bash scripts. Do you have an example?

Comment (seanpmorgan) [reply update_release_version.sh]: https://github.com/tensorflow/addons/blob/master/tools/install_deps/buildifier.sh as an example. You're right this is missing from most of our bash scripts... but should still add it I suppose.

**Pull request merged**